functions/go/format/README.md

```diff
@@ -0,0 +1,78 @@
+# format
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Format the field ordering in resources.
+
+<!--mdtogo-->
+
+### Synopsis
```

Comment:
General note: A command "synopsis" refers to the exact syntax used to describe a CLI. e.g.

https://kpt.dev/reference/pkg/get/

We're not using this word correctly in function usage. I suggest getting rid of Synopsis section and just having:

- Overview
- FunctionConfig
- Examples

Please follow up in a separate PR and change the template in the process.

Reply:
https://github.com/GoogleContainerTools/kpt/issues/1884